define/types.go

```diff
@@ -284,12 +277,6 @@ func cloneToDirectory(url, dir string) ([]byte, string, error) {
 		return combinedOutput, gitSubdir, fmt.Errorf("failed while performing `git remote add`: %w", err)
 	}
 
-	if gitRef != "" {
```

Comment:
There is no need for this `fetch` automatically resolves correct `tag`, `branch` or `commit`. If not found it fails.